cmd/misc/main.go

```diff
@@ -162,6 +163,17 @@ func main() {
 		IndexOldEth1Blocks(opts.StartBlock, opts.EndBlock, opts.BatchSize, opts.DataConcurrency, opts.Transformers, bt)
 	case "update-aggregation-bits":
 		updateAggreationBits(rpcClient, opts.StartEpoch, opts.EndEpoch, opts.DataConcurrency)
+	case "historic-prices-export":
```

Comment:
This does not include any logging. Starting the tool, especially now that after each day a delay of 5s has been added, will not give any feedback which might seem odd.

Consider adding stuff like
* Exporting day X
* Finished exporting day X, took Ys

Since this would add more code, I suggest to also move whole block to a separate function (like done for `updateAggreationBits` for example).

Reply:
You currently have logs like 
msg="fetching historic prices for day 2021-03-24 00:00:00 +0000 UTC"
for each day.
I added a general "exporting days X - X" at the start and a "finished export" after each day.

I also changed the "incomplete prices" to an error return so that we stop the export.